### PR TITLE
Add function for injecting event spikes into existing event recordings to serve as augmentation

### PR DESCRIPTION
**Pull request by NeelayS**

### CODE REVIEW

File: event_aug/utils.py [hunk 1]
```diff
@@ -90,10 +91,104 @@ def imgs_to_video(
...
+
+    if return_fps:
+        return arr, fps
+
+    return arr
+
+
 def save_video_frames_diffs(
     video_path: str, save_path: str, out_fps: int = None, neg_diff: bool = False
 ):
```
Comment (jamesturner246) [inline event_aug/utils.py:176]: It looks like negative differences are not considered by default. Was this intentional? Would be good to have pos and neg events by default.

**Review by jamesturner246 — COMMENTED** [review]
Looks good and nicely commented. I would like to test it on my machine also, just to make sure the code does as intended. Could you please send a link to an example video to convert, and the command for doing so? Else all looks good. Thanks.

File: event_aug/utils.py [hunk 2]
```diff
@@ -90,10 +91,104 @@ def imgs_to_video(
...
+    vid.release()
+
+    if return_fps:
+        return arr, fps
+
+    return arr
+
+
 def save_video_frames_diffs(
     video_path: str, save_path: str, out_fps: int = None, neg_diff: bool = False
```
[the remark this answers is not in the document]
Comment (NeelayS) [reply event_aug/utils.py:174]: Got it. I'll make the changes to this and the encoding functions as well.

[the remark this answers is not in the document]
Comment (NeelayS) [reply event_aug/utils.py]: #26 e5ec380

[the remark this answers is not in the document]
Comment (jamesturner246) [reply event_aug/utils.py]: Thanks Neelay. I will wait to test it on my machine, and then approve if it all works.

**Pull request merged**